client/blocks/reader-full-post/index.jsx

```diff
@@ -77,21 +77,12 @@ export class FullPostView extends React.Component {
 
 	hasScrolledToCommentAnchor = false;
 
-	componentWillMount() {
-		const self = this;
-		asyncRequire( 'twemoji', twemoji => {
```

Comment:
I don't know how to measure bundle size changes. 🙂

Twemoji is around 7KB gzipped by itself, I don't know if it's any better than that in the bundle. The bulk of Twemoji is in the images, which are only downloaded as needed.